js/select2.js

```diff
@@ -46,12 +46,10 @@
         if (Object.prototype.hasOwnProperty.call(config, 'ajax')) {
```

Comment:
This check is not sufficient since it applies sortable on single value select2 fields, like uid field (author).

Reply:
Added a check to ensure we use a multiple field.